features/fixtures/mazerunner/app/src/main/java/com/bugsnag/android/mazerunner/MainActivity.kt

```diff
@@ -91,6 +91,11 @@ class MainActivity : Activity() {
         }
     }
 
+    // As per JSONObject.getString but returns and empty string rather than throwing if not present
+    private fun getStringSafely(jsonObject: JSONObject, key: String): String {
+        return if (jsonObject.has(key)) jsonObject.getString(key) else ""
+    }
```

Comment:
This function is effectively what `JSONObject.optString` does. Included a suggestion below.